<ui version="4.0" >
 <class>AddIMContactWidget</class>
 <widget class="QWidget" name="AddIMContactWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>119</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Add contact</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="1" column="1" >
    <widget class="QLineEdit" name="contactIdLineEdit" >
     <property name="toolTip" >
      <string>Id of the contact to add (e.g: wengo@voip.wengo.fr)</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="toolTip" >
      <string>Id of the contact to add (e.g: wengo@voip.wengo.fr)</string>
     </property>
     <property name="text" >
      <string>Contact Id:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QComboBox" name="accountComboBox" >
     <property name="toolTip" >
      <string>Choose account to add the contact to</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="toolTip" >
      <string>Choose account to add the contact to</string>
     </property>
     <property name="text" >
      <string>Account: </string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="addButton" >
       <property name="text" >
        <string>Add</string>
       </property>
       <property name="autoDefault" >
        <bool>true</bool>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>accountComboBox</tabstop>
  <tabstop>contactIdLineEdit</tabstop>
  <tabstop>addButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
